{"uid":"8370d95dcdd0f921","name":"User able to login into QuickBase application","fullName":"testCases.test_login.TestHome#test_login_QuickBase","historyId":"21595260a12447d8b9a29b8affc045ee","time":{"start":1760502515697,"stop":1760502528077,"duration":12380},"status":"failed","statusMessage":"AssertionError: Typing failed in 'Username' after 2 attempts: Locator.wait_for: Timeout 5000ms exceeded.\nCall log:\n  - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible","statusTrace":"self = <pageObjects.login_page.LoginPage object at 0x7ff4ce96d7d0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/action/appoverview/4a037ed7-41fa-4f7b-98e4-19e436c73066'> selector='[name=\"loginid\"]'>\ntext = 'arunagowda984@gmail.com', field_name = 'Username', timeout = 5000\nretries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n>                   element = self._wait_for_element(locator, timeout)\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\npageObjects/BasePage.py:55: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects/BasePage.py:26: in _wait_for_element\n    element.wait_for(state=\"visible\", timeout=timeout)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/sync_api/_generated.py:17975: in wait_for\n    self._sync(self._impl_obj.wait_for(timeout=timeout, state=state))\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_locator.py:692: in wait_for\n    await self._frame.wait_for_selector(\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_frame.py:369: in wait_for_selector\n    await self._channel.send(\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:69: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7ff4ce860e50>\ncb = <function Channel.send.<locals>.<lambda> at 0x7ff4ce986ac0>\nis_internal = False, title = None\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False, title: str = None\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(\n            task, \"__pw_stack__\", None\n        ) or inspect.stack(0)\n    \n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal, title)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 5000ms exceeded.\nE           Call log:\nE             - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:558: TimeoutError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <testCases.test_login.TestHome object at 0x7ff4cfd424d0>\n\n    @allure.title(\"User able to login into QuickBase application\")\n    def test_login_QuickBase(self):\n>       LoginPage(self.page).login_into_quickbase()\n\ntestCases/test_login.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects/login_page.py:37: in login_into_quickbase\n    self.type(self.username_field, self.username, \"Username\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pageObjects.login_page.LoginPage object at 0x7ff4ce96d7d0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/action/appoverview/4a037ed7-41fa-4f7b-98e4-19e436c73066'> selector='[name=\"loginid\"]'>\ntext = 'arunagowda984@gmail.com', field_name = 'Username', timeout = 5000\nretries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n                    element = self._wait_for_element(locator, timeout)\n                    element.fill(text)\n                    return\n                except Exception as e:\n                    if attempt < retries:\n                        time.sleep(1)\n                        attempt += 1\n                    else:\n>                       raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\nE                       AssertionError: Typing failed in 'Username' after 2 attempts: Locator.wait_for: Timeout 5000ms exceeded.\nE                       Call log:\nE                         - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible\n\npageObjects/BasePage.py:63: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[{"name":"base_url","time":{"start":1760502310679,"stop":1760502310679,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"_verify_url","time":{"start":1760502310680,"stop":1760502310680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"pytestconfig","time":{"start":1760502310680,"stop":1760502310680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"delete_output_dir","time":{"start":1760502310680,"stop":1760502310680,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"setup_and_teardown","time":{"start":1760502496895,"stop":1760502497299,"duration":404},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"log_on_failure","time":{"start":1760502515697,"stop":1760502515697,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: Typing failed in 'Username' after 2 attempts: Locator.wait_for: Timeout 5000ms exceeded.\nCall log:\n  - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible","statusTrace":"self = <pageObjects.login_page.LoginPage object at 0x7ff4ce96d7d0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/action/appoverview/4a037ed7-41fa-4f7b-98e4-19e436c73066'> selector='[name=\"loginid\"]'>\ntext = 'arunagowda984@gmail.com', field_name = 'Username', timeout = 5000\nretries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n>                   element = self._wait_for_element(locator, timeout)\n                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n\npageObjects/BasePage.py:55: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects/BasePage.py:26: in _wait_for_element\n    element.wait_for(state=\"visible\", timeout=timeout)\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/sync_api/_generated.py:17975: in wait_for\n    self._sync(self._impl_obj.wait_for(timeout=timeout, state=state))\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_locator.py:692: in wait_for\n    await self._frame.wait_for_selector(\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_frame.py:369: in wait_for_selector\n    await self._channel.send(\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:69: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Connection object at 0x7ff4ce860e50>\ncb = <function Channel.send.<locals>.<lambda> at 0x7ff4ce986ac0>\nis_internal = False, title = None\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False, title: str = None\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(\n            task, \"__pw_stack__\", None\n        ) or inspect.stack(0)\n    \n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal, title)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.wait_for: Timeout 5000ms exceeded.\nE           Call log:\nE             - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible\n\n/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/playwright/_impl/_connection.py:558: TimeoutError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <testCases.test_login.TestHome object at 0x7ff4cfd424d0>\n\n    @allure.title(\"User able to login into QuickBase application\")\n    def test_login_QuickBase(self):\n>       LoginPage(self.page).login_into_quickbase()\n\ntestCases/test_login.py:10: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npageObjects/login_page.py:37: in login_into_quickbase\n    self.type(self.username_field, self.username, \"Username\")\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pageObjects.login_page.LoginPage object at 0x7ff4ce96d7d0>\nlocator = <Locator frame=<Frame name= url='https://team.quickbase.com/nav/app/bvhcuf7ef/action/appoverview/4a037ed7-41fa-4f7b-98e4-19e436c73066'> selector='[name=\"loginid\"]'>\ntext = 'arunagowda984@gmail.com', field_name = 'Username', timeout = 5000\nretries = 1\n\n    def type(self, locator, text: str, field_name: str = None, timeout=5000, retries=1):\n        \"\"\"Type text (auto-waits, supports string or Locator, retries once on failure).\"\"\"\n        step_name = field_name if field_name else locator\n        display_text = self.mask_sensitive(step_name, text)\n        with allure.step(f\"Type value '{display_text}' into {step_name} input field\"):\n            attempt = 0\n            while attempt <= retries:\n                try:\n                    element = self._wait_for_element(locator, timeout)\n                    element.fill(text)\n                    return\n                except Exception as e:\n                    if attempt < retries:\n                        time.sleep(1)\n                        attempt += 1\n                    else:\n>                       raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\nE                       AssertionError: Typing failed in 'Username' after 2 attempts: Locator.wait_for: Timeout 5000ms exceeded.\nE                       Call log:\nE                         - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible\n\npageObjects/BasePage.py:63: AssertionError","steps":[{"name":"Login into QuickBase application","time":{"start":1760502515698,"stop":1760502528076,"duration":12378},"status":"failed","statusMessage":"AssertionError: Typing failed in 'Username' after 2 attempts: Locator.wait_for: Timeout 5000ms exceeded.\nCall log:\n  - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.13/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/quickBase/quickBase/pageObjects/login_page.py\", line 37, in login_into_quickbase\n    self.type(self.username_field, self.username, \"Username\")\n  File \"/home/runner/work/quickBase/quickBase/pageObjects/BasePage.py\", line 63, in type\n    raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\n","steps":[{"name":"Navigating to https://team.quickbase.com/nav/app/bvhcuf7ef/action/appoverview/4a037ed7-41fa-4f7b-98e4-19e436c73066","time":{"start":1760502515698,"stop":1760502516914,"duration":1216},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"Type value 'arunagowda984@gmail.com' into Username input field","time":{"start":1760502517071,"stop":1760502528076,"duration":11005},"status":"failed","statusMessage":"AssertionError: Typing failed in 'Username' after 2 attempts: Locator.wait_for: Timeout 5000ms exceeded.\nCall log:\n  - waiting for locator(\"[name=\\\"loginid\\\"]\").first to be visible\n\n","statusTrace":"  File \"/home/runner/work/quickBase/quickBase/pageObjects/BasePage.py\", line 63, in type\n    raise AssertionError(f\"Typing failed in '{field_name}' after {retries + 1} attempts: {e}\")\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":2,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"log_on_failure::0","time":{"start":1760502528226,"stop":1760502528294,"duration":68},"status":"passed","steps":[],"attachments":[{"uid":"b1fb0cf04eaa3cb2","name":"failed_test","source":"b1fb0cf04eaa3cb2.png","type":"image/png","size":111890}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true},{"name":"setup_and_teardown::0","time":{"start":1760502528294,"stop":1760502528348,"duration":54},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"parentSuite","value":"testCases"},{"name":"suite","value":"test_login"},{"name":"subSuite","value":"TestHome"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"3583-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"testCases.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"f97a5fef7982cab","status":"failed","statusDetails":"AssertionError: Validation failed: Expected Simple Project Manager, got Project Manager Home","time":{"start":1760502497300,"stop":1760502513611,"duration":16311}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"8370d95dcdd0f921.json","parameterValues":[]}